### PR TITLE
Hotfix/7.1.2

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

**Review by mcornaton — APPROVED** [review]
Thanks! :)

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,5 @@
 version=7.1.1
```
Comment (jeremyjams) [inline gradle.properties:1]:
```suggestion
version=7.1.2
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+All notable changes to this project will be documented in this file.
+
+## [[7.1.1]](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/releases/tag/v7.1.1) 2022-07-01
+
+* Hotfix integration tests
+
+## 7.1.0 2022-07-01
+
+* Not released on GitHub
```
Comment (jeremyjams) [inline CHANGELOG.md:11]:
```suggestion
## [[7.1.0]](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/releases/tag/v7.1.0) 2022-07-01 
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,41 @@
-# iexec-blockchain-client
-A blockchain adapter API for iExec Smart-Contracts and other Ethereum things
+# iexec-blockchain-adapter-api
+
+## Overview
+
+A blockchain adapter API for iExec Smart-Contracts and other Ethereum things.
```
Comment (jeremyjams) [inline README.md:5]:
```suggestion
A blockchain adapter API for interacting with iExec Smart-Contracts and for doing other Ethereum things.
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,41 @@
...
+
+The iExec Blockchain Adapter API is available as an OCI image on [Docker Hub](https://hub.docker.com/r/iexechub/iexec-blockchain-adapter-api/tags).
+
+To run properly, the iExec Blockchain Adapter API requires:
+* A reachable blockchain node URL hosting iExec smart contracts.
+* A reachable MongoDB instance to persist its data.
+* A reachable broker to match iExec orders.
+* An Ethereum wallet to interact with Smart Contracts on the blockchain network.
```
Comment (jeremyjams) [inline README.md:13]:
```suggestion
To run properly, the iExec Blockchain Adapter API requires:
* A blockchain node. iExec smart contracts must be deployed on the blockchain network.
* A MongoDB instance to persist its data.
* A broker service to match iExec orders.
* An Ethereum wallet to interact with Smart Contracts on the blockchain network.
```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,41 @@
...
+
+## Configuration
+
+| Environment variables | Description | Values | Default value |
```
Comment (jeremyjams) [inline README.md:17]:
```suggestion
| Environment variable | Description | Type | Default value |
```

Comment (jbern0rd) [reply README.md]: Thanks, I missed those updates in the last round

File: README.md (hunk 4 above)
Comment (jbern0rd) [reply README.md]: :+1:

File: gradle.properties (hunk 1 above)
Comment (jbern0rd) [reply gradle.properties]: done in ef6e53b

**Review by jeremyjams — APPROVED** [review]

**Pull request merged**